browser.js

```diff
@@ -240,7 +240,11 @@ function init() {
 
 	// link the theme if it was changed while the app was closed
 	if (process.platform === 'darwin') {
-		storage.set('darkMode', osxAppearance.isDarkMode());
+		const isDarkMode = osxAppearance.isDarkMode();
```

Comment:
> OS X Dark Mode is not the same as app Dark Mode

I realize that. The changes you introduced here solve the issue but introduce another change: If I change my system preference it changes `anatine`. 1) it wasn't in the original issue, and 2) that doesn't seem like a reasonable expectation. Why should `anatine` change because I changed the color of the dock and menu bar?